packages/apps-config/src/ui/colors.ts

```diff
@@ -115,7 +117,8 @@ export const chainColors: Record<string, any> = [
   ['Galois', chainGalois],
   ['ZΞRO Alphaville', chainZero],
   ['Zenlink PC1', chainZenlink],
-  ['Uniarts', chainUniarts]
+  ['Uniarts', chainUniarts],
+  ['SubDAO PC1', chainSubDAO]
```

Comment:
Would have preferred this and the next one alphabetical, but will fix up afterwards - seems I was not awake on some and the order has gone slightly messy above.

Reply:
I fix it right now to end up the mess.

https://github.com/polkadot-js/apps/pull/4602